setup.py

```diff
@@ -57,6 +57,7 @@
         # Type stubs
         "pandas-stubs",
         "networkx-stubs",
+        "scipy-stubs",
         "types-docutils",
         "types-setuptools",
         "vivarium",
```

Comment:
This isn't really a typing stub? We really need to have vivarium imported since we use real `Artifact` objects.